test/remote/gateways/remote_mercado_pago_test.rb

```diff
@@ -84,6 +84,15 @@ def test_successful_purchase_with_processing_mode_gateway
     assert_equal 'accredited', response.message
   end
 
+  def test_successful_purchase_with_payment_method_option_id
```

Comment:
Again, because this new parameter is so closely tied into the existing fields for processing mode, I would suggest not writing a new test to cover this test case but to instead expand the existing `test_successful_purchase_with_processing_mode_gateway` by adding the new `payment_method_option_id` to the `@processing_options` hash.

Reply:
007ab496410a32775742ba9d92258cbc61952732

I've added `payment_method_option_id` to the `@processing_options` hash, but with one caveat, it's value. I believe it's real-word value would a hash, would it be worth just creating a random hash for this test?

Reply:
https://github.com/activemerchant/active_merchant/blob/e0395f63e520c902caa2985b5dd7523ee19ead38/test/remote/gateways/remote_mercado_pago_test.rb#L46